src/Oro/BugBundle/Tests/Unit/Entity/IssueStatusTest.php

```diff
@@ -22,7 +22,17 @@ public function getEntityFQCN()
      */
     public function getSetDataProvider()
     {
-        return ['label' => ['label', 'label', 'label']];
+        return [
```

Comment:
/**
     * @return string
     */
    public function __toString()
    {
        return $this->getLabel() ? $this->getLabel() : "";
    }

please refactor this code in entity